lldb/tools/lldb-dap/BreakpointBase.h

```diff
@@ -9,21 +9,27 @@
 #ifndef LLDB_TOOLS_LLDB_DAP_BREAKPOINTBASE_H
 #define LLDB_TOOLS_LLDB_DAP_BREAKPOINTBASE_H
 
-#include "llvm/Support/JSON.h"
 #include <string>
 
+#include "llvm/Support/JSON.h"
+
+#include "DAPForward.h"
```

Comment:
This should follow LLVM's [include style](https://llvm.org/docs/CodingStandards.html#include-style). Applies to all the other files in this PR as well.

Reply:
Updated all the files I touched in the CL to match the style.

Reply:
I ran `include-what-you-use`[^1] on the breakpoint implementation files and fixed the `#includes` for them. If you think that should go into a separate pull request, I can move that out of here and have a follow up for that as well.

[^1]: https://include-what-you-use.org/

Reply:
It would have been better off as a separate PR, particularly as this PR contains some not completely trivial refactors in lldb-dap.cpp. I'll just note that the llvm style does not include (for the most part) blank lines between the include groups.